src/lumigo_tracer/event/event_trigger.py

```diff
@@ -87,6 +102,20 @@ def _parse_http_method(event: dict):
     return _parse_http_method_v1(event)
 
 
+def _parse_load_balancer_method(event: dict):
+    result = {
+        "triggeredBy": "load_balancer",
+        "httpMethod": event.get("httpMethod", ""),
+        "resource": event.get("resource", ""),
+        "messageId": event.get("requestContext", {}).get("requestId", ""),
+    }
+    if isinstance(event.get("headers"), dict):
+        result["api"] = event["headers"].get("host")
+    if isinstance(event.get("requestContext"), dict):
+        result["stage"] = event["requestContext"].get("stage")
```

Comment:
can you try to create an elb/alb with stage?
I don't know if this is possible...